packages/http/src/getHttpOperations.ts

```diff
@@ -19,23 +19,20 @@ const httpAndFileResolver = new Resolver({
 });
 
 export async function getHttpOperationsFromResource(file: string): Promise<IHttpOperation[]> {
-  const fileContent = /^https?:\/\//i.exec(file)
-    ? (await axios.get(file, { transformResponse: res => res })).data
-    : fs.readFileSync(file, { encoding: 'utf8' });
+  const isRemote = /^https?:\/\//i.test(file);
+  const fileContent = isRemote ? await fetch(file).then(d => d.text()) : fs.readFileSync(file, { encoding: 'utf8' });
```

Comment:
It's quite hard to get on which part `await` has an effect. On the other hand, it looks pretty. Please extract it to separate func or ignore the comment if you also think it's pretty ;)

Sidenote: Pre-C programmers would not believe how many complicated things are possible using a single line of code.